templates/al2/provisioners/install-worker.sh

```diff
@@ -272,9 +272,30 @@ elif [ "$BINARY_BUCKET_REGION" = "eu-isoe-west-1" ]; then
 elif [ "$BINARY_BUCKET_REGION" = "us-isof-south-1" ]; then
   S3_DOMAIN="csp.hci.ic.gov"
 fi
+
+# TODO: start deprecating these.
 S3_URL_BASE="https://$BINARY_BUCKET_NAME.s3.$BINARY_BUCKET_REGION.$S3_DOMAIN/$KUBERNETES_VERSION/$KUBERNETES_BUILD_DATE/bin/linux/$ARCH"
 S3_PATH="s3://$BINARY_BUCKET_NAME/$KUBERNETES_VERSION/$KUBERNETES_BUILD_DATE/bin/linux/$ARCH"
 
+function s3_binary_path() {
+  local binary_name=""
+  local os="linux"
+  local arch="$ARCH"
+  local version=""
+  while getopts 'n:o:a:v:' OPTION; do
```

Comment:
using this in a function is a bit odd IMO. positional args would be more verbose in some places, but simpler to grok

Reply:
i agree 😅 looking for something with same defaults that wasn't too verbose, ill either keep iterating on it or just inline all the paths

Reply:
im removing it and deferring writing some helper until we migrate other dependencies